package.json

```diff
@@ -46,45 +46,44 @@
   },
   "homepage": "https://github.com/linode/design-language-system#readme",
   "devDependencies": {
-    "@storybook/addon-essentials": "^8.1.10",
-    "@storybook/addon-interactions": "^8.1.10",
-    "@storybook/addon-links": "^8.1.11",
-    "@storybook/blocks": "^8.1.10",
-    "@storybook/react": "^8.1.10",
-    "@storybook/react-vite": "^8.1.10",
-    "@storybook/test": "^8.1.10",
-    "@storybook/theming": "^8.1.10",
+    "@storybook/addon-essentials": "^8.3.0",
+    "@storybook/addon-interactions": "^8.3.0",
+    "@storybook/addon-links": "^8.3.0",
+    "@storybook/blocks": "^8.3.0",
+    "@storybook/react": "^8.3.0",
+    "@storybook/react-vite": "^8.3.0",
+    "@storybook/test": "^8.3.0",
+    "@storybook/theming": "^8.3.0",
```

Comment:
I wonder if we could only use one set of storybook dependencies between CM & DLS. (and maybe a couple other packages). DLS is here to stay now, so it could potentially server as a sole provide for some deps (I would only consider devDependencies)

Reply:
Yea would be a benefit if we maintain the monorepo approach where we can have one package.json hoisted that's shared across packages.

Reply:
👀 

Created an internal ticket [M3-8596] for investigating this!